hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -466,13 +466,31 @@
    */
   protected String useThisHostnameInstead;
 
+  /**
+   * @deprecated
+   *  Use {@link HRegionServer#UNSAFE_RS_HOSTNAME_DISABLE_MASTER_REVERSEDNS_KEY} instead.
+   */
+  @Deprecated
+  @InterfaceAudience.LimitedPrivate(HBaseInterfaceAudience.CONFIG)
+  final static String DEPRECATED_RS_HOSTNAME_DISABLE_MASTER_REVERSEDNS_KEY =
```

Comment:
Why not stick with the previous constant name?

Reply:
Have seen this kind of convention at couple of places in this repo - https://github.com/apache/hbase/blob/01974385642ea5672542b1d4a22fcb54e8e2ba02/hbase-common/src/main/java/org/apache/hadoop/hbase/io/ByteBuffAllocator.java#L81
https://github.com/apache/hbase/blob/025ddce868eb06b4072b5152c5ffae5a01e7ae30/hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/throttle/CompactionThroughputControllerFactory.java#L43